ocs_ci/utility/tnf_hypervisor.py

```
@@ -0,0 +1,937 @@
+            logger.info("No disk resizing needed")
+            return
+
+        cluster_name = self.dev_scripts_config.get("cluster_name", "tnf-cluster")
+        num_masters = self.dev_scripts_config.get("num_masters", 2)
+
+        for i in range(num_masters):
+            domain = f"{cluster_name}_master_{i}"
+            for dev_name, target_size in self._disk_resize_map.items():
+                logger.info(f"Resizing {dev_name} on {domain} to {target_size}G")
+                self._ssh_cmd(
+                    f"sudo virsh blockresize {domain} {dev_name} " f"{target_size}G"
```

Ruff (0.16.1)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)